packages/SwingSet/src/types-external.js

```diff
@@ -123,7 +123,9 @@ export {};
  * @typedef {[tag: 'error', problem: string]} DeviceInvocationResultError
  * @typedef { DeviceInvocationResultOk | DeviceInvocationResultError } DeviceInvocationResult
  *
- * @typedef { { d: VatDeliveryObject, syscalls: VatSyscallObject[] } } TranscriptEntry
+ * @typedef { { d: VatSyscallObject, response: VatSyscallResult } } TranscriptSyscall
```

Comment:
Yeah, the old type claimed that `syscalls` was a list of VSOs, but `transcript.js` actually stored a list of `{ d: VSO, response: VSR }`.

The new type names `TranscriptSyscall` as a pair VSO and a VSR (although the newer commit changes their names to be `{s, r}`, both to make them shorter, and because `d` is used for the vat *delivery* object in the enclosing object).